source/dev-setup.rst

```diff
@@ -137,14 +125,10 @@ you wish)::
 Creating a Python Virtual Environment
 '''''''''''''''''''''''''''''''''''''
 
-You can create a virtual environment using a version of Python installed
-from pythonz as follows::
+You can create a virtual environment using a version of Python as follows::
 
-    mkvirtualenv -p $(pythonz locate 3.4.4) my-virtualenv-name
+    mkvirtualenv -p $(which python) my-virtualenv-name
```

Comment:
I prefer using pyenv's own virtualenv plugin to handle my virtual environments.

https://github.com/pyenv/pyenv-virtualenv